src/ucp/proto/proto_common.h

```diff
@@ -54,6 +54,8 @@ typedef struct {
                                               minimal size of a single fragment */
     ptrdiff_t               max_frag_offs; /* offset in uct_iface_attr_t of the
                                               maximal size of a single fragment */
+    ptrdiff_t               max_iov_offs;  /* offset in uct_iface_attr_t of the
```

Comment:
could you explain pls? from first reading I thought that `max_iov_offset` is something like `max_iov_size - 1`...